apps/brandfolder/src/index.js

```diff
@@ -0,0 +1,87 @@
+import { setup } from 'shared-dam-app';
+
+const CTF_APP_URL = 'https://integration-contentful-app.brandfolder-svc.com'
```

Comment:
For the public marketplace app, the hosting will be done by us at https://brandfolder.ctfapps.net

Reply:
Updating, thanks!